src/StreamingDataFrames/quixstreams/dataframe/dataframe.py

```diff
@@ -235,6 +235,22 @@ def producer(self) -> RowProducerProto:
     def producer(self, producer: RowProducerProto):
         self._real_producer = producer
 
+    @staticmethod
+    def contains(key: str) -> Column:
+        """
+        Returns a Column configured to assess whether a specified key is present in rows.
+
+        :param key: A string representing the column name to check for in each row.
+        :return: A Column object configured to evaluate the presence of a column.
```

Comment:
Could we maybe re-phrase it a bit to make it shorter?

Like:
"
Check if the key is present in the `Row` value.

:param key: a column name to check
:returns: a `Column` object that evaluates to `True` if the key is present or `False` otherwise.
"

The examples are great 👍